data/Kabit Savaiye Bhai Gurdas Ji/235.json

```
@@ -39,7 +39,7 @@
         "pronunciation_information": null,
         "type": "Pankti",
         "gurmukhi": {
-          "Seva Singh": "jYsy mnu prwDIn hIn dInqw mY; swDsMg siqgur syvw n lgwveI ["
+          "Seva Singh": "jYso mnu lwgY prwDIn hIn dInqw mY, swD sMg siqgur syvw n lgwveI ]"
```

```suggestion
          "Seva Singh": "jYso mnu lwgY prwDIn hIn dInqw mY; swD sMig siqgur syvw n lgwveI ]"
```

Same reason as above. And `sMg` to `sMig` to match source.